config/software/scandir.rb

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright:: Copyright (c) 2013-2014 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "scandir"
+default_version "1.2"
+
+dependency "python"
+dependency "pip"
+
+build do
+  license "https://raw.githubusercontent.com/benhoyt/scandir/v1.2/LICENSE.txt"
```

Comment:
The DSL method that we currently use to include the license is `ship_license` instead of `license` (its usage is identical).

Could you replace it here? Thanks!